{{cookiecutter.project_name}}/README.md

```diff
@@ -102,36 +174,46 @@ bonfire namespace reserve
 ephemeral-cv17hi
 ```
 
-#### Building and pushing the backend starter application image
+Save the ephemeral namespace name for later, you can export it to an environment variable named `NAMESPACE` for
+simplicity.
+
+### Building and pushing the backend starter application image
 
-In order to deploy the backend starter app to an Ephemeral namespace, the user requires to push an image to
-an external registry accesible from the Ephemeral cluster.
+Make sure to log in to your Quay.io account before this step, this guide will asume you have created a repository in
+Quay to host your images. We **encourage you to use a public repository** to simplify the process, since a private
+repository will require you to create an additional secret on the Ephemeral namespace to be able to deploy the image.
 
 Make sure you're logged in against Quay using the `login` command of either `podman` or `docker` with write
 permissions against the target image repository.
 
-```
+```shell
+# podman
+podman login quay.io
+
+# docker
 podman login quay.io
```

Comment:
I think this should be `docker login quay.io`

Reply:
whops ... thank you, fixed!